ocs_ci/ocs/platform_nodes.py

```diff
@@ -0,0 +1,106 @@
+from ocs_ci.framework import config
+from ocs_ci.utility import aws
+
+
+class PlatformNodesFactory:
+    """
+    A factory class to get specific nodes platform object
+
+    """
+    def __init__(self):
+        self.cls_map = {'AWS': AWSNodes, 'VMWare': VMWareNodes}
+
+    def get_nodes_platform(self):
+        platform = config.ENV_DATA['platform']
+        return self.cls_map[platform]()
+
+
+class PlatformNodes(object):
+
+    def get_data_volume(self, node):
+        pass
+
+    def stop_nodes(self, nodes):
+        pass
+
+    def start_nodes(self, nodes):
+        pass
+
+    def restart_nodes(self, nodes):
```

Comment:
Is this 'reboot' of nodes?

Reply:
yes. I kept consistency with aws.restart_instances